python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_sensors.py

```diff
@@ -525,6 +544,41 @@ def test_start_sensor_with_default_status(self, graphql_context):
 
         assert start_result.data["startSensor"]["sensorState"]["status"] == "RUNNING"
 
+    def test_instigate_sensor(self, graphql_context):
+        instigator_selector = infer_sensor_selector(graphql_context, "always_no_config_sensor")
+        result = execute_dagster_graphql(
+            graphql_context,
+            EVALUATE_SENSOR_MUTATION,
+            variables={"selectorData": instigator_selector, "cursor": "blah"},
+        )
+        assert result.data
+        assert result.data["evaluateSensor"]["__typename"] == "SensorExecutionData"
+        assert result.data["evaluateSensor"]["cursor"] == "blah"
+        assert result.data["evaluateSensor"]["runRequests"] is None
+        assert result.data["evaluateSensor"]["skipMessage"] is None
+
+    def test_instigation_failure(self, graphql_context):
+        instigator_selector = infer_sensor_selector(graphql_context, "always_error_sensor")
+        result = execute_dagster_graphql(
+            graphql_context,
+            EVALUATE_SENSOR_MUTATION,
+            variables={"selectorData": instigator_selector, "cursor": "blah"},
+        )
+        assert result.data
+        assert result.data["evaluateSensor"]["__typename"] == "PythonError"
+        assert "Error occurred during the execution of evaluation_fn" in result.data["evaluateSensor"]["message"]
+
+    def test_instigation_non_existent_sensor(self, graphql_context):
+        instigator_selector = infer_sensor_selector(graphql_context, "sensor_doesnt_exist")
+        result = execute_dagster_graphql(
+            graphql_context,
+            EVALUATE_SENSOR_MUTATION,
+            variables={"selectorData": instigator_selector, "cursor": "blah"},
+        )
+        assert result.data
+        assert result.data["evaluateSensor"]["__typename"] == "SensorNotFoundError"
+        assert result.data["evaluateSensor"]["sensorName"] == "sensor_doesnt_exist"
+
```

Comment:
test a sensor that returns a cursor and ensure it doesn't progress on multiple evaluations